tools/azure_controller.sh

```diff
@@ -53,6 +51,8 @@ print_system_info() {
 
 print_system_info
 
+# enable core dumps
+sudo ulimit -c unlimited
```

Comment:
what does this have to do with core dumps?

Reply:
https://stackoverflow.com/questions/28982396/failed-to-write-core-dump-core-dumps-have-been-disabled

I guess the default core dump size is too small or even 0.